src/super_gradients/training/models/kd_models/kd_module.py

```diff
@@ -0,0 +1,62 @@
+from super_gradients.training.models.sg_module import SgModule
+import torch
+from super_gradients.training.utils.utils import HpmStruct
+
+
+class KDOutput:
+    def __init__(self, student_output: torch.Tensor = None, teacher_output: torch.Tensor = None):
```

Comment:
is it wise to have a default value for student_output? shouldn't it be obligatory?

Reply:
Will fix it.

Reply:
fixed